modules/contact-form/css/editor-ui.css

```diff
@@ -1,18 +1,26 @@
 i.mce-i-grunion {
-	font-family: dashicons;
 	font-size: 20px;
 }
 
 i.mce-i-grunion:before,
 .jetpack-contact-form-icon:before {
-	content: "\f175";
+	width: 24px;
 	vertical-align: top;
+	content: '';
+	display: block;
+	height: 24px;
+	background-size: 24px;
+	background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill="rgb(85, 93, 102)" d="M13 7.5h5v2h-5zm0 7h5v2h-5zM19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 16H5V5h14v14zM11 6H6v5h5V6zm-1 4H7V7h3v3zm1 3H6v5h5v-5zm-1 4H7v-3h3v3z"/></svg>');
```

Comment:
Letting SVG to inherit `color` by using `fill="currentColor"` instead of `fill="rgb(85, 93, 102)"` might be more appropiate here, but don't let this hold off merging.

https://css-tricks.com/cascading-svg-fill-color/

Reply:
I just tried that and it doesn't seem to work for my guess it is because it is part of the url attribute. 
Thanks for the suggestion.